boards/stm32f3discovery/src/main.rs

```diff
@@ -713,6 +713,7 @@ pub unsafe fn reset_handler() {
         debug!("{:?}", err);
     });
 
+    chip.enable_watchdog();
```

Comment:
I don't think we want to enable the watchdog by default for any of our upstream boards.

Reply:
Should I comment or delete it outright?

Reply:
Commenting is good!